src/index.html

```diff
@@ -6,9 +6,72 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home-page" class="logo">
+        <img src="images/logo.png" alt="Moyo-logo">
+      </a>
+
+      <nav>
+        <ul class="header__nav">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
```

Comment:
```suggestion
        <ul class="header__nav nav">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
          </li>
```

the navigation can be not only in the header and can be the separate independent block

it is better to use more readable names here

Reply:
Thank you for advice, done.